summoner-cli/src/Summoner/Project.hs

```diff
@@ -296,7 +296,9 @@ doGithubCommands Settings{..} = do
             Just _ -> do
                 isHubSuccess <- runHub repo
                 if isHubSuccess
-                then "git" ["push", "-u", "origin", "master"]
+                then do
+                    "git" ["push", "-u", "origin", "main"]
```

Comment:
I just googled and I see that the `git init` command now takes the argument `--initial-branch` to specify the default branch since `git 2.28`. I guess, we can add this option to our `git init` command above and update the README with the minimal `git` version required for the tool to work.

* Source: https://stackoverflow.com/questions/42871542/how-can-i-create-a-git-repository-with-the-default-branch-name-other-than-maste